src/App.tsx

```diff
@@ -3,36 +3,40 @@ import ActivitiesList from "@components/ActivitiesList/ActivitiesList";
 import { ACTIVITIES_MOCK } from "@components/ActivitiesList/mocks";
 import DataContainer from "@components/DataContainer/DataContainer";
 import HelloWorld from "@components/HelloWorld/HelloWorld";
-import { BrowserRouter, Route, Routes } from "react-router-dom";
+import { BrowserRouter, Link, Route, Routes } from "react-router-dom";
```

Comment:
nice use of `Link` here - people will usually try to make things like this a normal `a` tag, but that causes the whole page to reload at the new path, and defeats the purpose of having frontend routes and a Single Page App